aspnetcore/signalr/get-started-signalr-core/sample/Hubs/ChatHub.cs

```diff
@@ -9,7 +9,7 @@ public class ChatHub : Hub
         public Task Send(string user, string message)
         {
             string timestamp = DateTime.Now.ToShortTimeString();
-            return Clients.All.SendAsync(timestamp, user, message);
+            return Clients.All.SendAsync("Send", timestamp, user, message);
```

Comment:
We should change this string to `nameof(Send)`

Reply:
`nameof(Send)` implies that the server function is the same as the client function. It should be `"Send"`

Reply:
In that case, would it be a better idea to use a client function name other than "Send" in this example? This would prove that the names don't have to match.